docs/src/content/docs/it/guides/authoring-content.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Creazione di contenuti in Markdown
+description: Una panoramica della sintassi Markdown supportata da Starlight.
+---
+
+Starlight supporta l'intera sintassi [Markdown](https://daringfireball.net/projects/markdown/) nei file `.md` insieme al frontmatter [YAML](https://dev.to/paulasantamaria/introduction-to-yaml-125f) per definire metadati come il titolo e la descrizione.
+
+Assicurarsi di guardare la [documentazione MDX](https://mdxjs.com/docs/what-is-mdx/#markdown) o la [documentazione Markdoc](https://markdoc.dev/docs/syntax) se si vogliono usare questi formati, dato che il supporto Markdown può variare.
+
+## Stili in linea
+
+Il testo può essere **grassetto**, _corsivo_, o ~~barrato~~.
+
+```md
+Il testo può essere **grassetto**, _corsivo_, o ~~barrato~~.
+```
+
+Puoi [aggiungere un link ad un'altra pagina](/fr/getting-started/).
+
+```md
+Puoi [aggiungere un link ad un'altra pagina](/fr/getting-started/).
+```
```

Comment:
Qua possiamo usare `/it/`:

````suggestion
Puoi [aggiungere un link ad un'altra pagina](/it/getting-started/).

```md
Puoi [aggiungere un link ad un'altra pagina](/it/getting-started/).
```
````